runtime/foucoco/src/lib.rs

```diff
@@ -1252,13 +1252,22 @@ impl currency::CurrencyConversion<currency::Amount<Runtime>, CurrencyId> for Cur
 
 parameter_types! {
 	pub const RelayChainCurrencyId: CurrencyId = XCM(0);
+	pub const GetWrappedCurrencyId: CurrencyId = CurrencyId::Stellar(Asset::AlphaNum4 {
```

Comment:
Should we add a comment that describes the asset? Like, why are we using this one.